### PR TITLE
style(forgot password): shorten focus ring

### LINKED ISSUE #6957: Shorten the focus ring around Forgot password link
I noticed that in the Password Settings section the focus ring around the Forgot password link is full width. Would be nice to have it fit the link, instead of act as a block.

CURRENT
image

PROPOSED
image

### PR DESCRIPTION
**Pull request by shreya99oak**
fixes #6957

### CODE REVIEW

**Comment by lmorchard** [comment]
Just noticed that you've also got PR #6981 open for this issue. Is this new PR meant to replace that one?

**Comment by shreya99oak** [comment]
> Just noticed that you've also got PR #6981 open for this issue. Is this new PR meant to replace that one?

Hey, yes it replaces that one

**Comment by lmorchard** [comment]
Hmm, I think there might be a commit for another issue mixed into this PR now - affecting spinner and modal size doesn't seem to be a part of #6957

**Comment by shreya99oak** [comment]
> Hmm, I think there might be a commit for another issue mixed into this PR now - affecting spinner and modal size doesn't seem to be a part of#6957

Oops,I'll drop that commit

File: app/styles/modules/_password-row.scss [hunk 1]
```diff
@@ -297,3 +297,9 @@ input[type='text'] {
     background-color: #bdbdbe;
   }
 }
+
+//for the forgot password link
+a.reset-password {
```
Comment (lmorchard) [inline app/styles/modules/_password-row.scss]: Hmm, this kind of works, but I think we'd like the focus ring to shorten to the normal length of the link text.

Peeking in the inspector, it looks like the `display: flex` on `.links` causes the link to expand. You should be able to override that in this specific instance by adding `display: block` to [`.input-help-forgot-pw` here in `app/styles/modules/_input-row.scss`](https://github.com/mozilla/fxa-content-server/blob/a3c110e99b6c6d08da88b6d5e6d3e7de0ad0346d/app/styles/modules/_input-row.scss#L100)

Comment (shreya99oak) [reply app/styles/modules/_password-row.scss]: Okay, thanks!

**Comment by lmorchard** [comment]
Cool, looks like this works as a fix!

**Pull request merged**